unable to run local network with native linux zombienet

i am trying to spawn a relay + parachain network locally using the native linux zombienet binary (version `1.2.66`). the relay nodes seem to be spawned fine; however, my parachain doesnt spawn.

i think i am  missing something in the parachain settings. i dont even see any parachain logs created.

my network file (tried increasing the timeout value but no luck):
```
[settings]
timeout = 180000
default_args = [ "-lparachain=debug" ]

[relaychain]
default_command = "/home/abhi/code/polkadot/target/release/polkadot"
chain = "rococo-local"
chain_spec_path = "/home/abhi/code/launch-script/rococo-custom-2-raw.json"

  [[relaychain.nodes]]
  name = "alice"
  ws_port = 9988
  rpc_port = 30488

  [[relaychain.nodes]]
  name = "bob"
  ws_port = 9955
  rpc_port = 30555

[[parachains]]
id = 2001
addToGenesis = true
genesis_wasm_generator = "/home/abhi/code/ice-substrate/target/release/ice-node export-genesis-wasm --chain arctic-raw.json > arctic-genesis-wasm"
genesis_state_generator = "/home/abhi/code/ice-substrate/target/release/ice-node export-genesis-state --chain arctic-raw.json > arctic-genesis-state"

  [parachains.collator]
  name = "alice"
  command_with_args = "/home/abhi/code/ice-substrate/target/release/ice-node --ws-port=9944 --port=31300 --rpc-port=9933 --collator --tmp --chain=arctic-dev --alice --force-authoring --unsafe-ws-external --unsafe-rpc-external --rpc-external --ws-external --rpc-cors=all --rpc-methods=Unsafe --log=xcm=trace -- --chain=/home/abhi/code/ice-substrate/rococo-local-raw.json --execution=wasm"
  ws_port = 9944
  rpc_port = 9933
```

output:
```
↪ ./zombienet-linux -p native spawn zombienet-config.toml
╔════════════════════╤════════════════════════════════════════════════════════════════════════════════════════════════════╗
║ 🧟 Zombienet       │ Initiation                                                                                         ║
╟────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ Provider           │ native                                                                                             ║
╟────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ Namespace          │ zombie-2c132b6c4ca0fadf524919cc7b8106c5                                                            ║
╟────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ Temp Dir           │ /tmp/zombie-2c132b6c4ca0fadf524919cc7b8106c5_-368575-m3SRIMMbT86q                                  ║
╚════════════════════╧════════════════════════════════════════════════════════════════════════════════════════════════════╝

                 🚧 Chain Spec was set to a file in raw format, can't customize. 🚧

                 Chain name: Rococo Local Testnet

                 ⚙ Clear Boot Nodes
┌────────────────────┬────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ Launching          │ alice                                                                                              │
├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Command            │ /home/abhi/code/polkadot/target/release/polkadot --chain /tmp/zombie-2c132b6c4ca0fadf              │
│                    │ 524919cc7b8106c5_-368575-m3SRIMMbT86q/alice/cfg/rococo-local.json --name alice --rpc-cors          │
│                    │ all --unsafe-rpc-external --rpc-methods unsafe --unsafe-ws-external --no-mdns --node-key 2         │
│                    │ bd806c97f0e00af1a1fc3328fa763a9269723c8db8fac4f93af71db186d6e90 --no-telemetry --prometheu         │
│                    │ s-external --validator --prometheus-port 42779 --rpc-port 30488 --ws-port 9988 --listen-ad         │
│                    │ dr /ip4/0.0.0.0/tcp/42507/ws --base-path /tmp/zombie-2c132b6c4ca0fadf524919cc7b8106c5_-368         │
│                    │ 575-m3SRIMMbT86q/alice/data                                                                        │
├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Status             │ Ready                                                                                              │
└────────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────────┘
        alice running

                 You can follow the logs of the node by running this command:

                         tail -f  /tmp/zombie-2c132b6c4ca0fadf524919cc7b8106c5_-368575-m3SRIMMbT86q/alice.log

                 ⚙ Added Boot Nodes:  /ip4/127.0.0.1/tcp/42507/ws/p2p/12D3KooWQCkBm1BYtkHpocxCwMgR8yjitEeHGx8spzcDLGt2gkBm
┌────────────────────┬────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ Launching          │ bob                                                                                                │
├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Command            │ /home/abhi/code/polkadot/target/release/polkadot --chain /tmp/zombie-2c132b6c4ca0fadf              │
│                    │ 524919cc7b8106c5_-368575-m3SRIMMbT86q/bob/cfg/rococo-local.json --name bob --rpc-cors all          │
│                    │ --unsafe-rpc-external --rpc-methods unsafe --unsafe-ws-external --no-mdns --node-key 81b63         │
│                    │ 7d8fcd2c6da6359e6963113a1170de795e4b725b84d1e0b4cfd9ec58ce9 --no-telemetry --prometheus-ex         │
│                    │ ternal --validator --bootnodes /ip4/127.0.0.1/tcp/42507/ws/p2p/12D3KooWQCkBm1BYtkHpocxCwMg         │
│                    │ R8yjitEeHGx8spzcDLGt2gkBm --prometheus-port 44579 --rpc-port 30555 --ws-port 9955 --listen         │
│                    │ -addr /ip4/0.0.0.0/tcp/45677/ws --base-path /tmp/zombie-2c132b6c4ca0fadf524919cc7b8106c5_-         │
│                    │ 368575-m3SRIMMbT86q/bob/data                                                                       │
├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Status             │ Ready                                                                                              │
└────────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────────┘
        bob running

                 You can follow the logs of the node by running this command:

                         tail -f  /tmp/zombie-2c132b6c4ca0fadf524919cc7b8106c5_-368575-m3SRIMMbT86q/bob.log
         All relay chain nodes spawned...
Error: Timeout(60) for node : alice-1
    at NativeClient.<anonymous> (/snapshot/zombienet/dist/providers/native/nativeClient.js)
    at Generator.next (<anonymous>)
    at fulfilled (/snapshot/zombienet/dist/providers/native/nativeClient.js)

         Node's logs are available in /tmp/zombie-2c132b6c4ca0fadf524919cc7b8106c5_-368575-m3SRIMMbT86q/logs
```

Hello @pepoviola I think I get the same error as @abhijeetbhagat trying to launch an acala node. Here are the logs I've gathered:
<details><summary> Zombienet logs </summary>
```
 zombie creating node: acala1 +9s
  zombie::cmdGenerator adding -lsystem=debug,parachain=debug,runtime=debug,sc_basic_authorship=trace,cumulus-consensus=trace,cumulus-collator=trace,collator_protocol=trace,collation_generation=trace,aura=debug,pallet-collator-staking=trace,xcm=trace,pallet-xcm=trace +9s
  zombie::cmdGenerator Added --port with value 60149 +2ms
  zombie::cmdGenerator Added --ws-port with value 60150 +0ms
  zombie::cmdGenerator Added --rpc-port with value 60151 +0ms
  zombie::native::client {
  zombie::native::client     "metadata": {
  zombie::native::client         "name": "acala1",
  zombie::native::client         "namespace": "zombie-b3e555ea3e177022b21d6a4537c34c57",
  zombie::native::client         "labels": {
  zombie::native::client             "zombie-role": "cumulus-collator",
  zombie::native::client             "app": "zombienet",
  zombie::native::client             "zombie-ns": "zombie-b3e555ea3e177022b21d6a4537c34c57",
  zombie::native::client             "name": "zombie-b3e555ea3e177022b21d6a4537c34c57",
  zombie::native::client             "instance": "acala1"
  zombie::native::client         }
  zombie::native::client     },
  zombie::native::client     "spec": {
  zombie::native::client         "cfgPath": "/var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1/cfg",
  zombie::native::client         "dataPath": "/var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1/data",
  zombie::native::client         "ports": [
  zombie::native::client             {
  zombie::native::client                 "containerPort": 9615,
  zombie::native::client                 "name": "prometheus",
  zombie::native::client                 "flag": "--prometheus-port",
  zombie::native::client                 "hostPort": 60098
  zombie::native::client             },
  zombie::native::client             {
  zombie::native::client                 "containerPort": 9933,
  zombie::native::client                 "name": "rpc",
  zombie::native::client                 "flag": "--rpc-port",
  zombie::native::client                 "hostPort": 60097
  zombie::native::client             },
  zombie::native::client             {
  zombie::native::client                 "containerPort": 9944,
  zombie::native::client                 "name": "ws",
  zombie::native::client                 "flag": "--ws-port",
  zombie::native::client                 "hostPort": 60096
  zombie::native::client             },
  zombie::native::client             {
  zombie::native::client                 "containerPort": 30333,
  zombie::native::client                 "name": "p2p",
  zombie::native::client                 "flag": "--port",
  zombie::native::client                 "hostPort": 60095
  zombie::native::client             }
  zombie::native::client         ],
  zombie::native::client         "command": [
  zombie::native::client             "/Users/andrewplaza/Projects/acala/acala/target/production/acala --name acala1 --node-key fee029f0b50f1b22349ff24b815dac4250f42c149750a3e34b9df99c2ed07104 --chain /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1/cfg/rococo-local-2000.json --base-path /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1/data --listen-addr /ip4/0.0.0.0/tcp/60095/ws --rpc-port 60097 --ws-port 60096 --prometheus-external --prometheus-port 60098 --rpc-cors all --unsafe-rpc-external --rpc-methods unsafe --unsafe-ws-external --collator --force-authoring -lsystem=debug,parachain=debug,runtime=debug,sc_basic_authorship=trace,cumulus-consensus=trace,cumulus-collator=trace,collator_protocol=trace,collation_generation=trace,aura=debug,pallet-collator-staking=trace,xcm=trace,pallet-xcm=trace -- --chain /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1/cfg/rococo-local.json --execution wasm --port 60149 --ws-port 60150 --rpc-port 60151"
  zombie::native::client         ]
  zombie::native::client     }
  zombie::native::client } +2s
┌────────────────────┬────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ Pod                │ acala1                                                                                             │
├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Status             │ Launching                                                                                          │
├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Command            │ /Users/andrewplaza/Projects/acala/acala/target/production/acala --name acala1 --node-              │
│                    │ key fee029f0b50f1b22349ff24b815dac4250f42c149750a3e34b9df99c2ed07104 --chain /var/folders/         │
│                    │ 34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFn         │
│                    │ SNw6h/acala1/cfg/rococo-local-2000.json --base-path /var/folders/34/bwst5cqn00l_9vvz7tgbvs         │
│                    │ zc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1/data --liste         │
│                    │ n-addr /ip4/0.0.0.0/tcp/60095/ws --rpc-port 60097 --ws-port 60096 --prometheus-external --         │
│                    │ prometheus-port 60098 --rpc-cors all --unsafe-rpc-external --rpc-methods unsafe --unsafe-w         │
│                    │ s-external --collator --force-authoring -lsystem=debug,parachain=debug,runtime=debug,sc_ba         │
│                    │ sic_authorship=trace,cumulus-consensus=trace,cumulus-collator=trace,collator_protocol=trac         │
│                    │ e,collation_generation=trace,aura=debug,pallet-collator-staking=trace,xcm=trace,pallet-xcm         │
│                    │ =trace -- --chain /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022         │
│                    │ b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1/cfg/rococo-local.json --execution wasm --port          │
│                    │ 60149 --ws-port 60150 --rpc-port 60151                                                             │
└────────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────────┘
  zombie::native::client localFilePath /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/rococo-local.json +312ms
  zombie::native::client remoteFilePath /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/cfg/rococo-local.json +0ms
  zombie::native::client remote dir /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/cfg +0ms
  zombie::native::client data dir /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/data +0ms
  zombie::native::client localFilePath /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/zombie-wrapper.sh +9ms
  zombie::native::client remoteFilePath /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/cfg/zombie-wrapper.sh +0ms
  zombie::native::client remote dir /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/cfg +0ms
  zombie::native::client data dir /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/data +0ms
  zombie::native::client localFilePath /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala-dev-2000-rococo-local.json +0ms
  zombie::native::client remoteFilePath /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/cfg/rococo-local-2000.json +0ms
  zombie::native::client remote dir /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/cfg +0ms
  zombie::native::client data dir /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/data +0ms
  zombie::native::client /bin/bash +9ms
  zombie::native::client [
  zombie::native::client   '/Users/andrewplaza/Projects/acala/acala/target/production/acala --name acala1 --node-key fee029f0b50f1b22349ff24b815dac4250f42c149750a3e34b9df99c2ed07104 --chain /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1/cfg/rococo-local-2000.json --base-path /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1/data --listen-addr /ip4/0.0.0.0/tcp/60095/ws --rpc-port 60097 --ws-port 60096 --prometheus-external --prometheus-port 60098 --rpc-cors all --unsafe-rpc-external --rpc-methods unsafe --unsafe-ws-external --collator --force-authoring -lsystem=debug,parachain=debug,runtime=debug,sc_basic_authorship=trace,cumulus-consensus=trace,cumulus-collator=trace,collator_protocol=trace,collation_generation=trace,aura=debug,pallet-collator-staking=trace,xcm=trace,pallet-xcm=trace -- --chain /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1/cfg/rococo-local.json --execution wasm --port 60149 --ws-port 60150 --rpc-port 60151'
  zombie::native::client ] +0ms
  zombie::native::client 46648 +18ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +0ms
  zombie::native::client { exitCode: 0, stdout: '       0' } +29ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +24ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +77ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +59ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +62ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +63ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +58ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +47ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +39ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +50ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +46ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +48ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +51ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +51ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +49ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +27ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +49ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +50ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +28ms
  zombie::native::client [
  zombie::native::client   '-c',
  zombie::native::client   "grep -E 'Listening for new connections|Running JSON-RPC'  /var/folders/34/bwst5cqn00l_9vvz7tgbvszc0000gn/T/zombie-b3e555ea3e177022b21d6a4537c34c57_-46602-qAPLkFnSNw6h/acala1.log | wc -l"
  zombie::native::client ] +3s
  zombie::native::client { exitCode: 0, stdout: '       0' } +25ms
Error: Timeout(60) for node : acala1
    at NativeClient.<anonymous> (/snapshot/javascript/packages/orchestrator/dist/providers/native/nativeClient.js)
    at Generator.next (<anonymous>)
    at fulfilled (/snapshot/javascript/packages/orchestrator/dist/providers/native/nativeClient.js)
  zombie-cli TypeError: undefined is not a function
  zombie-cli     at /snapshot/javascript/packages/utils/dist/tableCli.js
  zombie-cli     at Array.forEach (<anonymous>)
  zombie-cli     at CreateLogTable.pushTo (/snapshot/javascript/packages/utils/dist/tableCli.js)
  zombie-cli     at CreateLogTable.pushToPrint (/snapshot/javascript/packages/utils/dist/tableCli.js)
  zombie-cli     at Network.<anonymous> (/snapshot/javascript/packages/orchestrator/dist/network.js)
  zombie-cli     at Generator.next (<anonymous>)
  zombie-cli     at fulfilled (/snapshot/javascript/packages/orchestrator/dist/network.js) +0ms
UnhandledRejection: TypeError: undefined is not a function
make: *** [run-native-full] Error 233
```
</details>

--- a/javascript/packages/orchestrator/src/cmdGenerator.ts
+++ b/javascript/packages/orchestrator/src/cmdGenerator.ts
@@ -47,7 +47,7 @@ export async function genCumulusCollatorCmd(
 
   // command with args
   if (commandWithArgs) {
-    return parseCmdWithArguments(commandWithArgs);
+    return parseCmdWithArguments(commandWithArgs, useWrapper);
   }
 
   const parachainAddedArgs: any = {

